### PR TITLE
Added itemTemplate property

### PR DESCRIPTION
**Pull request by Blackbaud-DeniseCisneros**
Added the `itemTemplate` property to the Text Expand Repeater docs

### CODE REVIEW

File: src/app/components/text-expand-repeater/index.html [hunk 1]
```diff
@@ -22,6 +22,12 @@
     >
     Specifies the number of items to display before truncating the list. This property accepts <sky-code>number</sky-code> values.
     </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="itemTemplate"
+      isOptional="true"
+    >
+    Specifies the template for items in the list. This property accepts <sky-code>any</sky-code> values.
```
Comment (Blackbaud-SteveBrush) [inline src/app/components/text-expand-repeater/index.html]: This should be `TemplateRef<any>`:
https://github.com/blackbaud/skyux-layout/blob/master/src/app/public/modules/text-expand-repeater/text-expand-repeater.component.ts#L45

Comment (blackbaud-johnly) [reply src/app/components/text-expand-repeater/index.html]: Should we link to https://angular.io/api/core/TemplateRef, @Blackbaud-SteveBrush? i.e., "This property accepts `TemplateRef<any>` values." where "`TemplateRef<any>` values" is a link to that Angular documentation?

File: src/app/components/text-expand-repeater/index.html [hunk 2]
```diff
@@ -22,6 +22,12 @@
     >
     Specifies the number of items to display before truncating the list. This property accepts <sky-code>number</sky-code> values.
     </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="itemTemplate"
+      isOptional="true"
+    >
+    Specifies the template for items in the list. This property accepts <sky-code>TemplateRef&lt;any></sky-code> values.
```
Comment (blackbaud-johnly) [inline src/app/components/text-expand-repeater/index.html]: I'd change "the" to "a" but other than that and Steve's feedback, this LGTM.

File: src/app/components/text-expand-repeater/index.html (hunk 1 above)
Comment (Blackbaud-SteveBrush) [reply src/app/components/text-expand-repeater/index.html]: Good idea! We're doing that for the inline-form `template` input as well.

Comment (Blackbaud-SteveBrush) [reply src/app/components/text-expand-repeater/index.html]: See: https://github.com/blackbaud/skyux2-docs/blob/master/src/app/components/inline-form/index.html#L23

**Pull request merged**